src/graph_ops.rs

```diff
@@ -204,7 +207,12 @@ fn split_by_partition<'a, Node: Idx, T>(
     splits
 }
 
-fn node_map_partition<Node, F>(node_map: F, node_count: Node, batch_size: usize) -> Vec<Range<Node>>
+fn greedy_node_map_partition<Node, F>(
```

Comment:
Why `greedy`? Some documentation maybe? :)

Reply:
Well, the construction algorithm is greedy. But yes, docs coming up!